macros/helpers/sqlserver/dropif.sql

```diff
@@ -0,0 +1,12 @@
+{% macro sqlserver__dropif(node) %}
```

Comment:
Could you make `dropif` (as defined in `macros/helpers/common.sql`) a "dispatched" macro, similar to the others, so that you can call this as simply `dropif`?

Reply:
will do!